src/tmpfiles/tmpfiles.c

```diff
@@ -2271,6 +2273,10 @@ static int process_item(Item *i) {
         r = arg_create ? create_item(i) : 0;
         q = arg_remove ? remove_item(i) : 0;
         p = arg_clean ? clean_item(i) : 0;
+        /* Failure can only be tolerated for create */
+        if (i->allow_failure) {
+                r = 0;
+        }
```

Comment:
please follow coding style: no {} around single-line if blocks, please. (See CODING_STYLE for details)